src/Forms/CopyLocaleAction.php

```diff
@@ -186,4 +186,25 @@ public function getGroup($gridField, $record, $columnName)
         }
         return null;
     }
+
+    /**
+     * Ensures that action is executed only once and not once per per locale
+     *
+     * @param string $actionName
+     * @param string $fromLocale
+     * @param string $toLocale
+     * @return bool
+     */
+    private function validateAction($actionName, $fromLocale, $toLocale)
+    {
+        if ($actionName === 'fluentcopyto' && $this->otherLocale === $fromLocale) {
```

Comment:
Hm I have a feeling this may be backwards.

```php
    /**
     * If true, this is "copy to $otherlocale". If false "copy from $otherLocale"
     *
     * @var bool
     */
    protected $isTo;
``` 

Let me test :)

Reply:
It would have worked, but you would end up with the `nz to us` action handling the `us to nz` event (still only one action though). I pushed up a fix to flip the from / to locales, and tested that it worked. :) I can't imagine there would be many cases it would effectively caused a bug, but at least the code is consistent.

Reply:
See https://github.com/tractorcow-farm/silverstripe-fluent/commit/81eff09f554551d8a9999c6c64f0286d18c5dbcb